x-pack/plugins/security_solution/public/management/pages/host_isolation_exceptions/constants.ts

```diff
@@ -15,6 +15,13 @@ import {
   ENDPOINT_HOST_ISOLATION_EXCEPTIONS_LIST_NAME,
 } from '@kbn/securitysolution-list-constants';
 
+export const SEARCHABLE_FIELDS: Readonly<string[]> = [
```

Comment:
👍 👍 
Will be re-used with the `ArtifactListPage` component when we refactor host isolation exceptions